src/components/Grid/Grid.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+
+import Grid, { GridCell } from './Grid';
+
+const GridExample = () => {
+  const columnCount: 1 | 2 | 3 = select('column count', [1, 2, 3], 3);
+
+  /**
+   * Creates an array of GridCell components based
+   * on the given columnCount from the Storybook select knob.
+   *
+   * @return {array} - array of React Components
+   */
+  const createGridCells = () => {
+    const cells = [];
```

Comment:
Juicy titbit for yer, if you wanna "func programming" it up you could also do

```
const cells = Array.from(Array(columnCount), (x, index) => {
      const i = index + 1;

      return (
        <GridCell column={i} columnCount={columnCount}>
          <div className="st-grid-cell-visualiser">
            {`Grid Cell ${i} of ${columnCount}`}
          </div>
        </GridCell>
      );
    });
```

😬
drop 🎤

Reply:
What do you think about `const i` not being type-checked? It throws this error:

```bash
Type 'number' is not assignable to type '1 | 2 | 3'
```

And trying to set explicit type on `i` (of `1 | 2 | 3`), e.g:

```javascript
const i: 1 | 2 | 3 = index + 1;
```

throws:

```bash
Type 'number' is not assignable to type '1 | 2 | 3'
```

Reply:
have you tried `const i: number = index + 1;` ?

Reply:
(realise we spoke about this IRL but just for others' clarity...) 

This doesn't work as the `column` property on `GridCell` is type-checked as `1 | 2 | 3`, so `i` must explicitly match that. 

I'll work on it a little more later.